client/src/featureform/resources.py

```diff
@@ -404,7 +404,7 @@ def serialize(self) -> bytes:
 class CassandraConfig:
     keyspace: str
     host: str
-    port: str
+    port: int
```

Comment:
might need to update `register_provider_test.py` those tests should flag this update